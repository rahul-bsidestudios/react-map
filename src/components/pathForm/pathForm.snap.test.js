import React from 'react';
import renderer from 'react-test-renderer';
import PathForm from './index';

const mockPath = jest.fn();
const mockClear = jest.fn();
const mockLoading = jest.fn();

test('renders PathForm component', () => {
  const tree = renderer.create(<PathForm createPath={mockPath} setLoading={mockLoading} clear={mockClear} />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders Pthform component with loading property', () => {
  const tree = renderer.create(<PathForm loading={true} createPath={mockPath} setLoading={mockLoading} clear={mockClear} />).toJSON();
  expect(tree).toMatchSnapshot();
});