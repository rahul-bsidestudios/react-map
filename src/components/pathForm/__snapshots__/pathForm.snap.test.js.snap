// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PathForm component 1`] = `
<div
  className="filters"
>
  <div
    className="form-group"
  >
    <label>
      Starting location
    </label>
    <div>
      <input
        autoComplete="off"
        className="form-control search react-mapbox-ac-input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Search"
        type="text"
        value=""
      />
      
      <span>
        <div
          className="react-mapbox-ac-menu"
          onClick={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
        >
          
        </div>
      </span>
    </div>
  </div>
  <div
    className="form-group last-control"
  >
    <label>
      Drop-off location
    </label>
    <div>
      <input
        autoComplete="off"
        className="form-control search react-mapbox-ac-input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Search"
        type="text"
        value=""
      />
      
      <span>
        <div
          className="react-mapbox-ac-menu"
          onClick={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
        >
          
        </div>
      </span>
    </div>
  </div>
  
  
  <div
    className="buttons"
  >
    <button
      className="btn btn-primary"
      onClick={[Function]}
      type="button"
    >
      Submit
    </button>
    <button
      className="btn btn-secondary"
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;

exports[`renders Pthform component with loading property 1`] = `
<div
  className="filters"
>
  <div
    className="form-group"
  >
    <label>
      Starting location
    </label>
    <div>
      <input
        autoComplete="off"
        className="form-control search react-mapbox-ac-input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Search"
        type="text"
        value=""
      />
      
      <span>
        <div
          className="react-mapbox-ac-menu"
          onClick={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
        >
          
        </div>
      </span>
    </div>
  </div>
  <div
    className="form-group last-control"
  >
    <label>
      Drop-off location
    </label>
    <div>
      <input
        autoComplete="off"
        className="form-control search react-mapbox-ac-input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Search"
        type="text"
        value=""
      />
      
      <span>
        <div
          className="react-mapbox-ac-menu"
          onClick={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
        >
          
        </div>
      </span>
    </div>
  </div>
  
  
  <div
    className="buttons"
  >
    <button
      className="btn btn-primary"
      disabled={true}
      onClick={[Function]}
      type="button"
    >
      Loading
    </button>
    <button
      className="btn btn-secondary"
      disabled={true}
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;
